packages/analytics-js/__fixtures__/fixtures.ts

```diff
@@ -394,6 +394,41 @@ const loadOptionWithInvalidEntry = {
   },
 };
 
+const anonymousIdWithNoStorageOption = {
```

Comment:
The rest of the entries can be omitted as they'll be defaulted to cookie storage, correct?

Reply:
This is the final storage state, not load option. I am updating the name.